code/datums/religions.dm

```diff
@@ -895,4 +895,18 @@
 	bible_names = list("How to Play Dwarf Fortress", "Book of Grudges", "Strike the Earth", "Lazy Newb Pack", "The Will of Armok", "Mining 101", "Hidden Fun Stuff and You")
 	male_adept = "Expedition Leader"
 	female_adept = "Expedition Leader"
-	keys = list("armok", "dwarf", "dorf", "dwarf fortress", "dorf fort")
\ No newline at end of file
+	keys = list("armok", "dwarf", "dorf", "dwarf fortress", "dorf fort")
+
+/datum/religion/art
+	name = "The Joy of Painting"
+	deity_name = "Bob Ross"
+	bible_names = list("The Joy of Painting")
```

Comment:
There's no need to use `bible_names` for a single bible, you can just set `bible_name = "The Joy of Painting"`